### ISSUE TITLE
Recipes for currently unobtainable resource seeds appearing in JEI/REI

### ISSUE THREAD

**Issue by FluffnTough** [BlakeBr0/MysticalAgriculture#605]
Apologies if this has already been addressed, however I have failed to find any reports of this specific topic.

I've built a little modpack for my friends and I and I have noticed that seed recipes for resources from mods that aren't currently installed are showing up in REI/JEI. Some examples include Uranium, Iridium, Chrome and Terrasteel, to name a few. 

Is this intentional? Would it be possible to make it so that resource seeds won't populate in REI/JEI if the mod necessary for a given resource is not installed? Or am I missing something?

Thanks in advance.

Minecraft: 1.19.2
Forge: 43.2.8 - also tested 43.2.14
Mystical Agriculture: 6.0.10 - also tested each version backwards to 6.0.7 as of time of writing.
Cucumber Lib: 6.0.7

**Comment by BlakeBr0** [comment]
Some of the modded crops aren't necessarily tied to a specific mod which is why they currently don't get hidden automatically. Terrasteel should only show up if Botania is installed, so I'm not entirely sure about that (assuming you don't have it installed).

As far as making those crops hide automatically, I've had issues trying to disable crops based on other factors besides mods loaded in the past, which is way I haven't done it. I may look into it at some point but it's not a high priority.

**Comment by FluffnTough** [comment]
I tested it with the other mods in the pack I'm working on (which I've named FluffnTough's Adventures, for reference) as well as in a control environment. In either case, Botania resource seeds and others do show.

> As far as making those crops hide automatically, I've had issues trying to disable crops based on other factors besides mods loaded in the past, which is way I haven't done it.

I think in previous MC versions, or in other modpacks I've played (not sure which is the deciding factor), it was tied into the Forge Ore dictionary, though I think that was in MC versions prior to 1.13. Not sure how it would work now as I believe the FOD was removed in favor of using the base game's Tags system.

> ...Terrasteel should only show up if Botania is installed, so I'm not entirely sure about that (assuming you don't have it installed).

I do not currently have Botania installed in either FA or the Control environment.

> I may look into it at some point but it's not a high priority.

Fair enough. I'm happy to at least know that it's known and I'm not just imagining things lol

Had I any experience in writing Java and with MC code specifically, I would offer to try to fix it.  I'd also like to note that with some seeds for resources like Platinum, for example, it shows for where the ingot would be: "Empty Tag: forge:ingots/platinum" with a red X where the image of the ingot would be. I've attached a screenshot for reference.

![Screenshot 2023-06-30 193121](https://github.com/BlakeBr0/MysticalAgriculture/assets/17916867/d9949d7b-1f7c-4ecf-8fd4-f7144da42b20)
